lib/helpbase.gi

```diff
@@ -214,7 +214,7 @@ if Length(positions) > 0 then # matches found
     begin := Minimum( patterns[i].finish+1, Length(topic) );
   od;
 
-  if begin < Length( topic ) then
+  if begin <= Length( topic ) then
```

Comment:
Please apply the same fix to `hpcgap/lib/helpbase.gi`

Reply:
O.K.